executor/standalone/standalone_test.go

```diff
@@ -1065,9 +1065,18 @@ func TestMetatron(t *testing.T) {
 		Version:         userSet.tag,
 		MetatronEnabled: true,
 		// The metatron test image writes out the task identity retrieved from the metadata service to `/task-identity`
-		EntrypointOld: fmt.Sprintf("/bin/bash -c \"cat /task-identity; grep %s /task-identity && grep jobAcceptedTimestampMs /task-identity | grep -E '[\\d+]'\"", t.Name()),
-		JobID:         generateJobID(t.Name()),
-		UsePodSpec:    shouldUsePodspecInTest,
+		EntrypointOld: strings.Join([]string{
```

Comment:
This was code that I didn't push as part of debugging metatron issues in https://github.com/Netflix/titus-executor/pull/637 - it's still useful for figuring out what went wrong, so I'll include it here.

Reply:
If you want you could use a backtick in the string to make it a true golang string literal and avoid quoting escapes, or indenting, or breaking it up at all (if you are OK hard-coding TestMetatron).

https://play.golang.org/p/SGFvIC02Oah